libmproxy/script.py

```diff
@@ -4,6 +4,8 @@
 import threading
 import shlex
 import sys
+from watchdog.observers import Observer
+from watchdog.events import PatternMatchingEventHandler, FileModifiedEvent
 
```

Comment:
1. Add a blinker event (as in [libmproxy.console.signals](https://github.com/mitmproxy/mitmproxy/blob/master/libmproxy/console/signals.py)) for script_change here.